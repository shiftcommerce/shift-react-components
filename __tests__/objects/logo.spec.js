// Libraries
import React from 'react'

// Object
import Logo from '../../src/objects/logo'

test('renders correctly', () => {
  // Arrange & Act
  const wrapper = mount(
    <Logo className='o-header__logo' />
  )

  // Assert
  expect(wrapper).toMatchSnapshot()
})
