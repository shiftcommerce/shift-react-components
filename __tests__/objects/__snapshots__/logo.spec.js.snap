// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Logo
  className="o-header__logo"
>
  <div
    className="o-header__logo"
  >
    <Link
      as="/"
      href="/slug?slug=/homepage"
    >
      <a
        as="/"
        href="/slug?slug=/homepage"
      >
        <Image
          height={40}
          src={Object {}}
          width={180}
        >
          <div>
            <picture>
              <img
                className="o-image"
                height={40}
                src={Object {}}
                srcSet="[object Object] 769w"
                width={180}
              />
            </picture>
          </div>
        </Image>
      </a>
    </Link>
  </div>
</Logo>
`;
